cmake/FindLibvterm.cmake

```diff
@@ -8,7 +8,17 @@ if(LIBVTERM_INCLUDE_DIR AND EXISTS "${LIBVTERM_INCLUDE_DIR}/vterm.h")
   file(STRINGS ${LIBVTERM_INCLUDE_DIR}/vterm.h VTERM_VERSION_MINOR REGEX "#define VTERM_VERSION_MINOR")
   string(REGEX MATCH "[0-9]+" VTERM_VERSION_MINOR ${VTERM_VERSION_MINOR})
 
-  set(VTERM_VERSION ${VTERM_VERSION_MAJOR}.${VTERM_VERSION_MINOR})
+  file(STRINGS ${LIBVTERM_INCLUDE_DIR}/vterm.h VTERM_VERSION_PATCH REGEX "#define VTERM_VERSION_PATCH")
+
+  # TODO(dundargoc): the following workaround is needed to give a coherent
+  # error for versions 0.3.2 and smaller. We can remove this once we anticipate
+  # that 0.3.2- becomes uncommon enough to not needing to account for it.
```

Comment:
Isn't it good to keep this in general?  That ensures `find_package(Libvterm 0.X.Y REQUIRED)` will always work. (What happened to bumping the min version to 0.3.3?)

Reply:
Sure, sounds good.

I reverted the bump to 0.3.3. I dunno, it didn't seem critical. I can rebump it if you think we should.

Reply:
OK, I'll rebump it then. @clason mind complementing the PR description with why a bump js needed? So I can use that for commit message. Bonus points for linking to corresponding issues.